src/lakefs_spec/client_helpers.py

```diff
@@ -329,6 +330,25 @@ def revert(client: LakeFSClient, repository: str, branch: str, parent_number: in
     )
 
 
+def reset_branch(client: LakeFSClient, repository: str, branch: str) -> None:
+    """
+    Reset the specified branch to its head.
+
+    Parameters
+    ----------
+    client: LakeFSClient
+        lakeFS client object.
+    repository: str
+        Repository in which the specified branch is located.
+    branch: str
+        Branch on which the commit should be reverted.
```

Comment:
Copy-paste, please update

Reply:
Fixed in 7706b7b3751020aefdc38bc9a0f9cde251435573